### ISSUE TITLE
CI failure: dynamic description not working?

### ISSUE THREAD

**Issue by stephane-caron** [stephane-caron/qpsolvers#77]
The CI fails with the following error:

```
        distutils.errors.DistutilsOptionError: No configuration found for dynamic 'description'.
        Some dynamic fields need to be specified via `tool.setuptools.dynamic`
        others must be specified via the equivalent attribute in `setup.py`.
        [end of output]
```

**Comment by stephane-caron** [comment]
Likely related to https://github.com/pypa/pip/issues/11294 I don't see the error with pip 20.0.2, but I'm getting it with:

```console
pip install pip==22.2 && pip install .
```

**Comment by stephane-caron** [comment]
Fixed in bb94e112ac384c8a3489767343f45a8ec3d5dc5a
